code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -66,6 +66,25 @@
 	var/medium_burn_msg = "blistered"
 	var/heavy_burn_msg = "peeling away"
 
+	/// The wounds currently afflicting this body part
+	var/list/wounds
+
+	/// The scars currently afflicting this body part
+	var/list/scars
+	/// Our current stored wound damage multiplier
+	var/wound_damage_multiplier = 1
```

Comment:
uh, do you use wound_damage_multiplier anywhere?

I can only find three mentions of it with a control+f search: one's here, one's in a comment, and one's just it getting set, not used